src/main/resources/io/jenkins/plugins/tuleap_api/Messages.properties

```diff
@@ -1,5 +1,6 @@
 TuleapNotifyCommitStatusStep.displayName = Update the build status of the commit in Tuleap
 TuleapNotifyCommitStatusStep.success = Success
 TuleapNotifyCommitStatusStep.failure = Failure
+TuleapNotifyCommitStatusStep.pending = Pending
```

Comment:
It seems you forgot to add the french translation in `Messages_fr.properties`

Reply:
Thanks!